packer_images/dead_letter_indexing_pipeline.conf.in

```diff
@@ -33,6 +33,6 @@ output {
   elasticsearch {
     hosts => ["${ES_DATA_NODES_URL}"]
     sniffing => false
-    index => ".logstash_dead_letter"
+    index => ".${DEAD_LETTER_PREFIX}_dead_letter"
```

Comment:
Since there will be a new DLQ name, I think we'll need to add it in the ES config to allow ES to dynamically create the index:
https://github.com/hmcts/ccd-elastic-search/blob/6db85a08b00519fe969ac4905fd27e59f57cb029/variables.tf#L62

Reply:
Thanks Dan, will raise an additional PR now and link it here

Reply:
https://github.com/hmcts/ccd-elastic-search/pull/37 PR to resolve @danlysiak comment